src/t8_geometry/t8_geometry_implementations/t8_geometry_examples.hxx

```diff
@@ -75,6 +75,13 @@ struct t8_geometry_squared_disk: public t8_geometry_with_vertices
     SC_ABORT_NOT_REACHED ();
   }
 
+  /* Load tree data is empty since we have no tree data.
+   * We need to provide an implementation anyways. */
+  void
+  t8_geom_load_tree_data (t8_cmesh_t cmesh, t8_gloidx_t gtreeid)
+  {
+    SC_ABORT_NOT_REACHED ();
+  }
```

Comment:
This should crash t8code. The load tree data function is executed every time the tree is loaded into memory

Reply:
Or it should at least crash the example

Reply:
This part of the code has not been developed by me. It possibly comes from a merge?

Reply:
Yes, apparently it was in main once, but after catching the bug it got removed:
https://github.com/DLR-AMR/t8code/commit/00b61eafd93a850e65b394c2136c31b11f3e5def
This is the second thing which tries to re-enter main via this PR. We definitely have to read everything very carefully so that nothing else tries to sneak in.